test/tab-components/tabPagesTest.js

```diff
@@ -90,10 +91,20 @@ describe('tab pages', function () {
 
     describe('tabs per page setting', function () {
       it('takes effect immediately', function * () {
-        const defaultTabsPerPage = appConfig.defaultSettings[settings.TABS_PER_PAGE]
+        const newValue = 6
+        const tabs = appConfig.defaultSettings[settings.TABS_PER_PAGE]
+        const numberOfPages = Math.ceil(tabs / newValue)
+        const numberOfTabs = tabs - (Math.floor(tabs / newValue) * newValue)
         yield this.app.client
-          .changeSetting(settings.TABS_PER_PAGE, 1)
-          .waitForElementCount(tabPage, defaultTabsPerPage)
+          .waitForBrowserWindow()
+          .loadUrl('about:preferences')
```

Comment:
Problem is that is not working for me. Most of the times we use if with 0, maybe 1. But my use case is that I need to focus last active tab, which is this case 19 (we have 20 tabs). But if do `.tabByIndex(19)` second tab is selected.